### DISCUSSION TITLE
Ethernet added

### DISCUSSION THREAD

**Discussion by timmbogner** [timmbogner/Farm-Data-Relay-System#137] [Show and tell]
I added ethernet yesterday. To use it, enable ```USE_ETHERNET```. In my case I only need to add ```#define ETH_CLK_MODE ETH_CLOCK_GPIO16_OUT``` somewhere before setup(), and then bring my ethernet power pin (GPIO5) high in setup() to make it work. 
@generalmanager I'm going to tag you when I commit my ethernet gateway example. Would you mind modifying my example to work with your Olimex boards?

**Comment by aviateur17** [comment]
@timmbogner, in fdrs_gateway_wifi.h line 93
`WiFi.onEvent(WiFiEvent);`
Should that be ETH.onEvent(ETHEvent);?
And then rename the callback on line 48 to ETHEvent?
